Microsoft.Identity.Web/ClaimPrincipalExtension.cs

```diff
@@ -107,10 +54,10 @@ public static string GetDomainHint(this ClaimsPrincipal claimsPrincipal)
         /// <remarks>See https://docs.microsoft.com/en-us/azure/active-directory/develop/id-tokens#payload-claims </remarks>
         public static string GetDisplayName(this ClaimsPrincipal claimsPrincipal)
         {
-            // Attempting the claims brought by an Microsoft identity platform token first
+            // Use the claims in an Microsoft identity platform token first
             string displayName = claimsPrincipal.FindFirstValue(ClaimConstants.PreferredUserName);
 
-            // Otherwise falling back to the claims brought by an Azure AD v1.0 token
+            // Otherwise fall back to the claims in an Azure AD v1.0 token
```

Comment:
Already fixed in the Microsoft.Identity.Web master source (in the ASP.NET Core Web app sample)